tests/support/peerManager.ts

```diff
@@ -164,9 +168,69 @@ export class RadiclePeer {
     this.spawn("radicle-node", args);
 
     await waitOn({
-      resources: [`tcp:${this.#listenSocketAddr}`],
-      timeout: 7000,
+      resources: [`socket:${Path.join(this.#radHome, "node", "control.sock")}`],
     });
+
+    this.rad(["node", "events"], { cwd: this.#radHome }).stdout?.on(
+      "data",
+      (data: any) => {
+        data
+          .toString()
+          .split("\n")
+          .forEach((data: unknown) => {
+            if (data && typeof data === "string" && data.trim() !== "") {
+              try {
+                const result: NodeEvent = JSON.parse(data);
+                this.#eventRecords.push(result);
+              } catch (e) {
+                console.log("Error parsing event", data);
+              }
+            }
+          });
+      },
+    );
+  }
+
+  public async routesTo(rid: string, ...nodes: string[]) {
```

Comment:
Could we call this function `addRoute`?

Reply:
I have no issue with renaming the function, but the goal is to wait for the routing table to have a specific entry.
So maybe we could rename it to `waitForRoutes` ? So it's similar to `waitForEvent`wdyt?

Reply:
Yeah, `waitForRoutes` makes more sense.

Reply:
e4c2861